code/Examples/C++/RPCClientServerBasic/CMakeLists.txt

```
@@ -0,0 +1,56 @@
+# Copyright 2025 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.20)
+
+project(RPCClientServerBasic)
```

I know that this implies changing all code, but you will need to regenerate it anyway when Fast DDS Gen PR is merged. 

```suggestion
project(RpcClientServerBasic)
```